src/alire/alire-config-edit.ads

```diff
@@ -22,6 +23,23 @@ package Alire.Config.Edit is
                   Value : String;
                   Check : CLIC.Config.Check_Import := null);
 
+   --  Typed alternatives
+
+   procedure Set_Boolean (Level : Config.Level;
```

Comment:
Are these used somewhere?

Reply:
Not yet, but there are old uses of `Set` that could be migrated